### PR TITLE
LPS-173358 Adding upgrades to normalize tables

### PR DESCRIPTION
**Pull request by achaparro**
LPS-173358-BUG-v1-db-upgrade-fix

### CODE REVIEW

**Comment by achaparro** [comment]
ci:reopen

File: modules/apps/portal/portal-dao-test/src/testIntegration/java/com/liferay/portal/dao/db/test/DBTest.java [hunk 1]
```diff
@@ -257,18 +263,47 @@ public void testAlterTableDropIndexedColumn() throws Exception {
...
+	}
 
-		Assert.assertFalse(_dbInspector.hasTable(_TABLE_NAME));
+	@Test
+	public void testGetPrimaryKeyColumnNamesIncorrectOrder() throws Exception {
+		_db.runSQL(
+			StringBundler.concat(
+				"create table ", _TABLE_NAME_2,
+				" (id1 LONG not null, id2 LONG not null, primary key (id1, ",
+				"id2))"));
```
Comment (drewbrokke) [inline modules/apps/portal/portal-dao-test/src/testIntegration/java/com/liferay/portal/dao/db/test/DBTest.java]: @achaparro I think in order to fully test this, we'll need to make sure that the expected primary keys are not in alphabetical order. Am I mis-reading this?

**Comment by brianchandotcom** [comment]
@drewbrokke @achaparro see changes in my origin. I may have broken some tests.

File: modules/apps/portal/portal-dao-test/src/testIntegration/java/com/liferay/portal/dao/db/test/DBTest.java (hunk 1 above)
Comment (achaparro) [reply modules/apps/portal/portal-dao-test/src/testIntegration/java/com/liferay/portal/dao/db/test/DBTest.java]: @drewbrokke You are right, I missed that. Fixed here:
https://github.com/brianchandotcom/liferay-portal/pull/131487

Thanks.

Comment (drewbrokke) [reply modules/apps/portal/portal-dao-test/src/testIntegration/java/com/liferay/portal/dao/db/test/DBTest.java]: Great, thanks @achaparro!

**Pull request closed without merging**